include/ci_nanovg.hpp

```diff
@@ -47,6 +48,7 @@ class Context {
 
   void path2dSegment(Path2d::SegmentType, const vec2 *p1, const vec2 *p2, const vec2 *p3);
 
+  std::string file;
```

Comment:
Looks like this does not need to be member data. It should just be a local variable in `Context::createFont()`.